11-parallelism/src/tests.rs

```diff
@@ -0,0 +1,59 @@
+#![allow(unused_mut, unused_variables, dead_code)]
+
+mod simple_thread {
+
+    use std::thread;
+
+    #[test]
+    fn spawn_a_thread_and_get_his_id() {
+        let father_id = thread::current().id();
+        let mut child_id = thread::current().id(); // The goal is to change that value
+
+        //TODO: spawn your thread here and return his id
+
+
+        //TODO: Update child_id with the thread::join() return value (id of the thread)
+
+        assert_ne!(father_id, child_id);
+    }
+
+    use std::{
+        sync::mpsc::{channel, Sender},
+        time::Duration,
+    };
+
+    #[test]
+    fn spawn_a_thread_and_use_rust_channel_to_communicate_between_threads() {
+        let message_to_send = "hello";
+        let (sender, receiver) = channel();
+
+
+        //TODO:
+        // - Spawn a new thread
+        // - Use the "sender" above to send the message `message_to_send` to your receiver
+
+
+        let msg_recv = receiver.recv_timeout(Duration::from_millis(20))
+                            .unwrap_or("nope");
```

Comment:
`unwrap_or` or just `expect` ?

Reply:
I use `unwrap_or` to pass the build step.

Reply:
Ok.